libraries/core_libs/network/src/tarcap/packets_handlers/common/ext_votes_packet_handler.cpp

```diff
@@ -17,7 +20,10 @@ void ExtVotesPacketHandler::onNewPbftVotes(std::vector<std::shared_ptr<Vote>> &&
 
     std::vector<std::shared_ptr<Vote>> send_votes;
     for (const auto &v : votes) {
-      if (!peer.second->isVoteKnown(v->getHash()) && peer.second->pbft_round_ <= v->getRound()) {
+      if (!peer.second->isVoteKnown(v->getHash()) &&
+          (peer.second->pbft_round_ <= v->getRound() ||
+           (v->getType() == cert_vote_type &&
+            v->getBlockHash() == pbft_mgr_->getLastPbftBlockHash() /* reward vote */))) {
```

Comment:
we do not need this check as this is called from `sendRewardVotes` and there can not be any other votes, exept ones that passed this condition. Or create separate function that will be used only for that and no round check are required